### PR TITLE
Add README.md for translation process

### PR DESCRIPTION
**Pull request by yuhattor**
@spier 
I have created a draft regarding the Translation Process. I may be missing some parts.

I'm a little confused, with the methodology you mentioned in #422, for example, when a Japanese community member sends a PR for a translation, should they send it to the book-jp branch? Or should they send PRs directly to main? From your description, it sounds like book-jp is just a place to put the artifacts. 
It would be good to indicate here clearly which branch to send PRs to when others join the translation in the future.

When the process is somewhat finalized, I will create the Japanese version in ```translation/japanese/README.md``` as well.

### CODE REVIEW

File: translation/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Translation Process and Recommendations
+
+If you would like to contribute to the InnerSource Patterns translation, great! We would love to work with you!
+First, please refer to [CONTRIBUTING.md](/CONTRIBUTING.md) for basic contribution instructions.
+This document covers basic translation principles, how you can contribute, and the review process. Please note that the content of this document is recommendations, mainly aimed at removing barriers for new contributors to the translation process. If you see something missing, please feel free to make suggestions.
```
Comment (spier) [inline translation/README.md]:
```suggestion

First, please refer to [CONTRIBUTING.md](/CONTRIBUTING.md) for basic contribution instructions.

This document covers basic translation principles, how you can contribute, and the review process. 

Please note that this document contains recommendations, aimed at removing barriers for new contributors to the translation process. If you see something missing, please feel free to make suggestions.
```

Adding some spacing to improve readablility.

File: translation/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## The Translation Principles and Recommendation
+
+- The target of the translation is structured patterns. You can find them at ```/patterns/2-structured/```, and only structured patterns are available in Gitbook.
```
Comment (spier) [inline translation/README.md]:
```suggestion
- The target of the translation are structured patterns. Find them in folder [/patterns/2-structured/](../patterns/2-structured/). Only structured patterns will be published in Gitbook.
```

Linking to the folder might make it even more clear which patterns we are talking about.

Also did some other rewording.

File: translation/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+- Please send your Pull Request.
+
+### How to translate new InnerSource Patterns
+
+- Please copy the file ```/patterns/2-structured/<file>.md``` to ```/translation/<language>/patterns/<file>.md``` and start translation.
+
+### How to Start Translating a New Language
+
+That is awesome! We welcome your passion. When implementing InnerSource practices in your region, there can be language barriers. There is great value in getting people in your organization to understand InnerSource in their own language. The translation should begin by creating a ```book-<YOUR_LANGUAGE>``` branch to work on. It is a good idea to name the branch after the abbreviation of the language. Currently, ```book-en``` is used for the English version and ```book-jp``` for the Japanese version.
```
Comment (spier) [inline translation/README.md]:
```suggestion
That is awesome! We welcome your passion. When implementing InnerSource practices in your region, there can be language barriers. There is great value in getting people in your organization to understand InnerSource in their own language. 

The translation should begin by creating a ```book-<YOUR_LANGUAGE>``` branch to work on. Please name the branch after the abbreviation of the language e.g. ```book-jp``` is used for the Japanese version.
```

The `book-en` is only used for testing right now, as I cannot merge to `main` yet. 
Therefore I simplified this sentence. Also added some spacing.

I really like the first paragraph here. It could even be moved to the very beginning of this doc, right after the title. It would be great to explain the motivation behind creating a translation of the pattterns in a bit more detail, as that might also motivate other translators to join.

One specific question about "getting people in your organization ...": 
Did you mean "region" here, or "organization"?

**Review by spier — COMMENTED** [review]
Thanks for putting this together! Looks great already.

I left some inline comments for you to consider.

Please make your own judgement calls about my inline comments, and then just merge whatever version of this doc you are comfortable with.

That also allows us to confirm that you indeed have WRITE permissions on the repo now :)

One cosmetic suggestion:
If you don't mind, could you use the code syntax with a single backtick, rather than 3 backticks? That's how we have done it elsewhere in this repo so far, so it would make it more consistent. (I am adding one inline suggestion about this to the doc but the comment applies to all code blocks).

Update:
Don't worry about the failing checks. 
I have taken care of them already.

File: translation/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+## How to Contribute to the Translation
+
+### How to Improve an Existing Translation
+
+- Please send your Pull Request.
+
+### How to translate new InnerSource Patterns
+
+- Please copy the file ```/patterns/2-structured/<file>.md``` to ```/translation/<language>/patterns/<file>.md``` and start translation.
```
Comment (spier) [inline translation/README.md]:
```suggestion
- Please copy the file `/patterns/2-structured/<file>.md` to `/translation/<language>/patterns/<file>.md` and start translation.
```

Using single backticks, to be consistent with the rest of the repo.

**Comment by spier** [comment]
> I'm a little confused, with the methodology you mentioned in #422, for example, when a Japanese community member sends a PR for a translation, should they send it to the book-jp branch? Or should they send PRs directly to main? From your description, it sounds like book-jp is just a place to put the artifacts.
> It would be good to indicate here clearly which branch to send PRs to when others join the translation in the future.

I can see how it can be confusing.

It will work like this from a **contributor perspective**:

- **scenario A: contributor wants to make a larger contribution** (like translating a new pattern or similar)
  - creates a PR with the changes against `main`
  - a trusted committer (TC) reviews the PR and eventually merges it (or rejects it, of course)
  - the PR is approved and merged
  - then the TC has to make sure that the changes on `main` are also added to `book-jp` so that they are visible in gitbook (this will happen manually in the beginning but we can automate it later)

- **scenario B: a contributor clicks somewhere on the JP gitbook on the "Edit on GitHub" link**
  - that will automatically create a PR agains the `book-jp` branch
  - the TC reviewing the PR then has to [change the target branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request) to `main`, to make sure that the changes end up in the mainline, and the contributions are attributed to the contributor correctly ("making the GH profile green").
  - then the same flow as in scenario A

I don't know if this is the best way to do this but it is the solution that I was able to come up with :)

To explain this in simpler terms we could say:
> All change should be created as PRs against the `main` branch

This is the default behavior in GitHub anyways.
The contributors coming from **scenario B** might not do this but should not be a big deal as the TC can do this for them.
Also at least for the EN book we did not have many people using the "Edit on GitHub" but who knows, maybe for the JP book it will be different :)

Does this help?

File: translation/README.md (hunk 1 above)
Comment (yuhattor) [reply translation/README.md]: Thanks, LGTM:)

File: translation/README.md (hunk 2 above)
Comment (yuhattor) [reply translation/README.md]: LGTM👍

File: translation/README.md (hunk 4 above)
Comment (yuhattor) [reply translation/README.md:22]: Got it!

File: translation/README.md (hunk 3 above)
Comment (yuhattor) [reply translation/README.md]: Thank you!
Actually, I meant to imply "organization" in this, but given the context, I now think "region" is more appropriate

File: translation/README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+- Please send your Pull Request.
+
+### How to translate new InnerSource Patterns
+
+- Please copy the file `/patterns/2-structured/<file>.md` to `/translation/<language>/patterns/<file>.md` and start translation.
+
+### How to Start Translating a New Language
+
+That is awesome! We welcome your passion. When implementing InnerSource practices in your region, there can be language barriers. There is great value in getting people in your organization to understand InnerSource in their own language. The translation should begin by creating a ```book-<YOUR_LANGUAGE>``` branch to work on. It is a good idea to name the branch after the abbreviation of the language. Currently, ```book-en``` is used for the English version and ```book-jp``` for the Japanese version.
```
Comment (yuhattor) [inline translation/README.md]:
```suggestion
That is awesome! We welcome your passion. When implementing InnerSource practices in your region, there can be language barriers. There is great value in getting people in your organization to understand InnerSource in their own language. 

The translation should begin by creating a `book-<YOUR_LANGUAGE>` branch to work on. Please name the branch after the abbreviation of the language e.g. `book-jp` is used for the Japanese version.
```

**Comment by yuhattor** [comment]
Thanks for your kind comments and onboarding:)
I have committed your suggestions and could verify that I had write permission

> One cosmetic suggestion:
If you don't mind, could you use the code syntax with a single backtick, rather than 3 backticks? That's how we have done it elsewhere in this repo so far, so it (I am adding one inline suggestion about this to the doc but the comment applies to all code blocks).

Got it, I will adopt single backtick from now on.


Also I understand regarding the branch. We are currently translating _Getting Started with InnerSource_, and thankfully many people are participating regarding this one. The text will be more accurate as more people review it. We are trying to create a policy in the Japanese community to translate InnerSource accordingly to ensure an accurate understanding of InnerSource. The book is very short and easy to read and would be a very good book for beginners, and that would be the same for community members and early contributors.

Once we have decided on that translation policy, we would like to apply it to InnerSource Patterns as well to make it more accurate in the future. If that happens, I think InnerSource Patterns will have more contributors.

There will be more modifications to the branch strategy in the future, but I'll keep an eye on it and try to accurately onboard other members. As for the branch strategy, I'll make sure to properly onboard the other members:)

File: translation/README.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+- Please send your Pull Request.
+
+### How to translate new InnerSource Patterns
+
+- Please copy the file `/patterns/2-structured/<file>.md` to `/translation/<language>/patterns/<file>.md` and start translation.
+
+### How to Start Translating a New Language
+
+That is awesome! We welcome your passion. When implementing InnerSource practices in your region, there can be language barriers. There is great value in getting people in your organization to understand InnerSource in their own language.
```
Comment (yuhattor) [inline translation/README.md]:
```suggestion
That is awesome! We welcome your passion. When implementing InnerSource practices in your region, there can be language barriers. There is great value in getting people in your organization to understand InnerSource in their own language.
```

Small fix for markdown lint

File: translation/README.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
+# Translation Process and Recommendations
+
+If you would like to contribute to the InnerSource Patterns translation, great! We would love to work with you!
+
+First, please refer to [CONTRIBUTING.md](/CONTRIBUTING.md) for basic contribution instructions.
+
+This document covers basic translation principles, how you can contribute, and the review process.
```
Comment (yuhattor) [inline translation/README.md]:
```suggestion
This document covers basic translation principles, how you can contribute, and the review process.
```

Small fix for markdown lint
